src/modules/Dropdown/Dropdown.js

```diff
@@ -994,10 +999,18 @@ export default class Dropdown extends Component {
     if (options === undefined || _.every(options, 'disabled')) return
 
     const lastIndex = options.length - 1
+    const { wrapSelection } = this.props
     // next is after last, wrap to beginning
     // next is before first, wrap to end
     let nextIndex = startIndex + offset
-    if (nextIndex > lastIndex) nextIndex = 0
+
+    /*
+    * if 'wrapSelection' is set and selection
+    * is after last or before first, it just does not change
+    */
```

Comment:
`//` for inline comments, please.  This should also fit on one line.  The linter allows 120 columns I believe.